### ISSUE TITLE
100% classification accuracy? 

### ISSUE THREAD

**Issue by kavnat** [charlesq34/pointnet#152]
Hello. I am using PointNet for a 3 class classification and I am getting 100% accurate classification for 2 classes and 99.5% for one. I am confused about the results and would like some input as to whether these results actually hold up. 

A little Background: I am using KITTI dataset for car instances, Polyterrasse dataset from University of Freiburg for Pedestrian instances, and ModelNet40 for miscellaneous dataset. The entire data has been downsampled to 256 points using uniform distribution. Not all the samples are being used, but each class has equal number of data samples in training and validation files. The data is split 7:3 training:validation. The accuracy and Loss curve on tensorboard plot look like the images attached. Can someone please help me understand if these numbers are valid? If they are, does it mean my network is overtrained? From what I understand if the network is overfitting the loss goes up and the accuracy crashes. In my case they both go stagnant after a few epochs of training. Have I made a mistake here? 
![acc_75](https://user-images.githubusercontent.com/38287725/48674191-c2509100-eb49-11e8-8a8c-7812bd0bc47d.PNG)
![loss_75](https://user-images.githubusercontent.com/38287725/48674197-c9779f00-eb49-11e8-8d40-2c02fd9473ba.PNG)

**Comment by OmarAhmadien** [comment]
Can you provide the graph for validation sets?
